support/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM hseeberger/scala-sbt:8u181_2.12.6_1.2.1
```

Comment:
If you're not adding any additional things to the `scala-sbt` image, I'd suggest just saying `image: hseeberger/scala-sbt:8u181_2.12.6_1.2.1` in your `docker-compose`. Also, have a look at [dfresh](https://github.com/realestate-com-au/dfresh) to lock down the exact image so everyone is guaranteed to have the same environment.